src/Kafker/Helpers/Emit.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Kafker.Configurations;
+using Kafker.Kafka;
+using McMaster.Extensions.CommandLineUtils;
+
+namespace Kafker.Helpers
+{
+    public class Emit : IEmitter
```

Comment:
Let's rename the class `Emit` to `SimpleEventsEmiter`?

Reply:
✓